willa/etl/doc_proc.py

```diff
@@ -0,0 +1,48 @@
+"""
+Utility functions to gather pdf files, use langchain pypdf loader to load them,
+and split them into chunks for vectorization.
+"""
+
+import os
+
+from langchain_community.document_loaders import PyPDFDirectoryLoader
+# from langchain_community.document_loaders import DirectoryLoader
+from langchain.text_splitter import RecursiveCharacterTextSplitter
+from langchain_core.vectorstores.base import VectorStore
+import willa.config
```

Comment:
I'm getting a pylint notification that I never use this import. is it auto included elsewhere? or should I alter line 17 to use it specifically?

Reply:
For the time being, you need to disable the PyLint warning.  I'm still wondering if there may be a better way.

```suggestion
import willa.config  # pylint: disable=W0611
```